tests/compile-fail/weak_memory/cpp20_rwc_syncs.rs

```diff
@@ -25,7 +25,7 @@ fn reads_value(loc: &AtomicUsize, val: usize) -> usize {
 fn static_atomic(val: usize) -> &'static AtomicUsize {
     let ret = Box::leak(Box::new(AtomicUsize::new(val)));
     // A workaround to put the initialization value in the store buffer.
```

Comment:
Can you open an issue for the problem of "pre-filling the store buffer when the first atomic access is a write", and reference that issue from this comment?